2016-2017_Starstruck/Module Prototypes/include/LCD.h

```diff
@@ -19,7 +21,7 @@ typedef struct MenuItem {
  * This function should be called from initialize()
  * DO NOT initialize the LCD separately
  */
-MenuItem *lcdmInit();
+MenuItem *lcdmInit(FILE *lcdPort);

```

Comment:
Not necessary the way you did it. Since all of the Frames including the root are in the scope of the recorderLoop function, they're not being passed around or accessed from more than one task. You just need to be careful about how you stop the recording (you'll probably have to use a boolean flag and serialize the list from within recorderLoop)